cmake/IgnUtils.cmake

```diff
@@ -1804,3 +1804,73 @@ macro(_ign_cmake_parse_arguments prefix options oneValueArgs multiValueArgs)
   endif()
 
 endmacro()
+
+#################################################
+# ign_generate_conf()
+# Generate conf/*.yaml files for projects
+macro(ign_generate_conf)
+    # Used only for internal testing.
+    set(ign_library_path "${CMAKE_BINARY_DIR}/test/lib/$<CONFIG>/ruby/ignition/cmd${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}")
+
+    # Generate a configuration file for internal testing.
+    # Note that the major version of the library is included in the name.
+    configure_file(
+      "${GZ_DESIGNATION}.yaml.in"
+      "${CMAKE_CURRENT_BINARY_DIR}/${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}.yaml.configured" @ONLY)
+
+    file(GENERATE
+      OUTPUT "${CMAKE_BINARY_DIR}/test/conf/$<CONFIG>/${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}.yaml"
+      INPUT "${CMAKE_CURRENT_BINARY_DIR}/${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}.yaml.configured")
+
+    # Used for the installed version.
+    set(ign_library_path "${CMAKE_INSTALL_PREFIX}/lib/ruby/ignition/cmd${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}")
+
+    # Generate the configuration file that is installed.
+    # Note that the major version of the library is included in the name.
+    configure_file(
+      "${GZ_DESIGNATION}.yaml.in"
+      "${CMAKE_CURRENT_BINARY_DIR}/${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}.yaml" @ONLY)
+    # Install the yaml configuration files in an unversioned location.
+
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR}.yaml
+      DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATAROOTDIR}/ignition/)
+
+endmacro()
+
+#################################################
+# ign_generate_cmd()
+# Generate cmd*.rb files for projects
+macro(ign_generate_cmd)
```

Comment:
Can you change the name of the macro to be a little more descriptive in the same way as my comment above?